crates/sui-snapshot/src/reader.rs

```diff
@@ -522,7 +522,7 @@ pub async fn download_bytes(
     part_num: &u32,
     max_timeout_secs: Option<u64>,
 ) -> (Bytes, [u8; 32]) {
-    let max_timeout = Duration::from_secs(max_timeout_secs.unwrap_or(30));
+    let max_timeout = Duration::from_secs(max_timeout_secs.unwrap_or(60));
```

Comment:
Do we need these timeout changes?

Reply:
We may not now, although these are for different things, and we may as well keep it. All the other timeouts in this PR are for a single reqwest connection, whereas this is the timeout across retries. From what I'm seeing, we're no longer needing to retry. That said, I would like to see if this helps on the R2 side, since the other changes above are specific to GCS and I wasn't yet able to find the corresponding settings on the AWS store builder. Will take a look at that later.